### DISCUSSION TITLE
BreakMeshByBlockGenerator interface

### DISCUSSION THREAD

**Discussion by renapf** [idaholab/moose#27551] [Q&A Modules: Solid mechanics]
Hello, what exactly differs when using the add_interface_on_two_sides = true, shouldn't it be giving the same results overall?

**Comment by GiudGiud** [comment]
Hello

The sideset is being added on both sides of the cut
As opposed to being only on one side 

It's usually more a matter of needing the two sidesets for some model that relies on both, for example a contact model.
If you only have one side, then you cant use the model

Guillaume
